qcodes/instrument_drivers/Keysight/Keysight_33500B_channels.py

```diff
@@ -253,21 +256,6 @@ def __init__(self, name, address, silent=False, **kwargs):
 
         super().__init__(name, address, **kwargs)
 
-        def errorparser(rawmssg):
```

Comment:
Same as above

Reply:
ok, i'll remove it.

Reply:
I won't remove it because it requires updating some example notebooks, i'd prefer to do that in a different PR.